src/eru.Domain/Entity/Class.cs

```diff
@@ -1,27 +1,34 @@
-﻿namespace eru.Domain.Entity
+﻿using System;
+
+namespace eru.Domain.Entity
 {
     public class Class
     {
-        public Class()
+        private Class()
         {
             //Required by EF
         }
 
-        public Class(string name)
+        public Class(int year, string section)
         {
-            Name = name;
+            Year = year;
+            Section = section;
```

Comment:
Strings should be normalized when constructor is called. 

```suggestion
            Section = section.Trim().ToLower();
```